src/perfetto_cmd/perfetto_cmd.cc

```diff
@@ -1039,6 +1039,11 @@ void PerfettoCmd::OnConnect() {
       args.clone_trigger_boot_time_ns = snapshot_trigger_info_->boot_time_ns;
       args.clone_trigger_delay_ms = snapshot_trigger_info_->trigger_delay_ms;
     }
+
+    if (trace_out_stream_) {
+      args.output_file_fd = base::ScopedFile(dup(fileno(*trace_out_stream_)));
```

Comment:
add a comment saying that here we have to apss the file anyways because we dont' know if the session is write_into_file or not. traced will decide ... yada yada